src/components/progressiveImage/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {View, StyleSheet, Animated} from 'react-native';
+import {AnimatedImage} from 'react-native-ui-lib'; //eslint-disable-line
```

Comment:
We import from `react-native-ui-lib` only in our example files 
in the src files always import using relative path to avoid cyclic requires.